propka/parameters.py

```diff
@@ -260,18 +267,18 @@ def print_interaction_parameters(self):
                                      group1, group2)[1]
                                  != 4)):
                         map_interaction += '?  '
-                info(interaction, map_interaction)
+                _LOGGER.info("%s %s", interaction, map_interaction)
                 if group1 == group2:
                     break
-            info('-')
-        info('--------------- Exceptions ----------------------------')
-        info('COO-HIS', self.COO_HIS_exception)
-        info('OCO-HIS', self.OCO_HIS_exception)
-        info('CYS-HIS', self.CYS_HIS_exception)
-        info('CYS-CYS', self.CYS_CYS_exception)
-
-        info('--------------- Mapping -------------------------------')
-        info("""
+            _LOGGER.info('-')
+        _LOGGER.info('--------------- Exceptions ----------------------------')
+        _LOGGER.info('COO-HIS %s', self.COO_HIS_exception)
+        _LOGGER.info('OCO-HIS %s', self.OCO_HIS_exception)
+        _LOGGER.info('CYS-HIS %s', self.CYS_HIS_exception)
+        _LOGGER.info('CYS-CYS %s', self.CYS_CYS_exception)
+
+        _LOGGER.info('--------------- Mapping -------------------------------')
+        _LOGGER.info("""
```

Comment:
This looks like latex, meant to be copied and pasted. Is this used by anyone? If so, perhaps make it easier to grep it from the logs by prefixing every line with some marker such as "LATEX>" or somesuch??

It might not be worth any effort, though. Up to you.

Reply:
You and @speleo3 know that I love to delete code... and I don't think anyone is using this.  I would happily delete if you both agree.

Reply:
It's the same to me – I am pretty sure I never looked at it.